menu.go

```diff
@@ -57,6 +57,8 @@ type MenuItem struct {
 	Icon Resource
 	// Since: 2.2
 	Shortcut Shortcut
+	// Since: 2.2.2
```

Comment:
We don't allow API changes in patch releases so this will need to be for 2.3.
This is also why we only use 2 digits for API level :)

Reply:
Done.